pkg/controller/pxcrestore/restore.go

```diff
@@ -24,13 +24,21 @@ func (r *ReconcilePerconaXtraDBClusterRestore) restore(cr *api.PerconaXtraDBClus
 		case bcp.Status.Destination[:4] == "pvc/":
 			return errors.Wrap(r.restorePVC(cr, bcp, bcp.Status.Destination[4:], cluster), "pvc")
 		case bcp.Status.Destination[:5] == "s3://":
-			return errors.Wrap(r.restoreS3(cr, bcp, bcp.Status.Destination[5:], cluster), "s3")
+			return errors.Wrap(r.restoreS3(cr, bcp, bcp.Status.Destination[5:], cluster, false), "s3")
 		}
 	}
 
 	return errors.Errorf("unknown destination %s", bcp.Status.Destination)
 }
 
+func (r *ReconcilePerconaXtraDBClusterRestore) pitr(cr *api.PerconaXtraDBClusterRestore, bcp *api.PerconaXtraDBClusterBackup, cluster api.PerconaXtraDBClusterSpec) error {
+	if cr.Spec.PITR != nil {
+		return errors.Wrap(r.restoreS3(cr, bcp, bcp.Status.Destination[5:], cluster, true), "PITR restore")
```

Comment:
why do we think that input will be with "s3://" prefix?
it can be any `some-bucket` or `s3://some-bucket` or `https://exmaple.com/some-bucket`